statistics/selectivity_test.go

```diff
@@ -889,3 +889,43 @@ func (s *testStatsSuite) TestRangeStepOverflow(c *C) {
 	// Must execute successfully after loading the column stats.
 	tk.MustQuery("select * from t where col between '8499-1-23 2:14:38' and '9961-7-23 18:35:26'").Check(testkit.Rows())
 }
+
+func (s *testStatsSuite) TestSmallRangeEstimation(c *C) {
+	defer cleanEnv(c, s.store, s.do)
+	testKit := testkit.NewTestKit(c, s.store)
+	testKit.MustExec("use test")
+	testKit.MustExec("drop table if exists t")
+	testKit.MustExec("create table t(a int)")
+	for i := 0; i < 1200; i++ {
+		testKit.MustExec(fmt.Sprintf("insert into t values (%v)", i/3)) // [0, 400)
+	}
```

Comment:
Better to combine them into one SQL like `insert into t values (.), (.), ...` to speed up.